files/install/host-cleaner.py

```diff
@@ -37,31 +38,38 @@ def foreman_wrapper(foreman_call, call_args=None):
     return result
 
 
-@baker.command()
-def clean_old_certificates(json_file=None):
+@baker.command(params={"check_on_fs": "Check on /var/lib/puppet/ssl/ca/signed/ to get the certificate list",
+                       "json_file": "Path to json file with the list on certificater to delete"})
+def clean_old_certificates(json_file=None, check_on_fs=False):
     """ This is a 'one time use' method that will clear all puppet cert for instances that doesn't still exist """
     # Retrieve config from ENV
     foreman_url = os.environ.get('FOREMAN_URL')
     foreman_user = os.environ.get('FOREMAN_USER')
     foreman_password = os.environ.get('FOREMAN_PASSWORD')
-    foreman_proxy_url = "https://{}:{}".format(os.environ.get('FOREMANPROXY_HOST'), os.getenv('FOREMANPROXY_PORT','8443'))
+    foreman_proxy_url = "https://{}:{}".format(os.environ.get(
+        'FOREMANPROXY_HOST'), os.getenv('FOREMANPROXY_PORT', '8443'))
 
     # connect to Foreman and ForemanProxy
-    f=Foreman(foreman_url, (foreman_user, foreman_password), api_version=2)
+    f = Foreman(foreman_url, (foreman_user, foreman_password), api_version=2)
     fp = ForemanProxy(foreman_proxy_url)
 
-    # Build a certificates list with only hostcert, discarding specific certs used for foreman, puppet, etc ...
-    host_pattern = ['ndev', 'nsta', 'nifd', 'npra', 'nifp-es5k', 'nhip', 'nifh', 'win', 'nprd']
-    if not json_file:
-        jcerts = fp.get_certificates().keys()
+    host_pattern = ['ndev', 'nsta', 'nifd', 'npra',
```

Comment:
tu ne semble pas avoir tout les combinaisons. Vite comme ca il manque nqa et nifp.

Reply:
Why is it on github? These prefix are used solely by Coveo AFAIK.

Reply:
- Au début le script n'était pas lié au fonctionnement de coveo, mais depuis le rajout du clean du DS il l'est. 
- `nifp` avait été volontairement exclu pour éviter de clear certain certificat comme celui de puppet `nifp-puppet1.prod.cloud.coveo.com`